packages/unity-react-core/.storybook/decorators.tsx

```diff
@@ -28,9 +28,10 @@ const StaticStory = ({args, Container, children, rootRef}) => {
     /**
      * Storybook only useId() will prefix the id with this identifier allowing
      * us to identify when the output is meant for bootstrap (non react)
-     */
+    */
     rootRef.current.innerHTML = renderToStaticMarkup(children, { identifierPrefix: identifierPrefix });
 
+    window.initDataLayer()
```

Comment:
This is inside the bootstrap decorator. This is how we log events with bootstrap so we need to initialize here.